superbuild/projects_modules/pyncpp.cmake

```diff
@@ -31,7 +31,7 @@ function(pyncpp_project)
 
         set(project_args
             GIT_REPOSITORY ${GITHUB_PREFIX}LIRYC-IHU/pyncpp.git
-            GIT_TAG working
+            GIT_TAG v0.1
```

Comment:
I did this in https://github.com/medInria/medInria-public/pull/1236 but I can close that PR. In any case it should be 0.1.x

Reply:
(updated for compilation on my side)